doc/ar65.sgml

```diff
@@ -32,16 +32,17 @@ for the cc65 compiler. ar65 is part of this suite.
 The archiver is called as follows:
 
 <tscreen><verb>
-   	Usage: ar65 <operation> lib file|module ...
-   	Operation is one of:
-   	       	a	Add modules
-   	       	d	Delete modules
-   	       	l	List library contents
-   	       	x	Extract modules
-       	       	V	Print the archiver version
+	Usage: ar65 <operation ...> lib file|module ...
+	Operations are some of:
+		r	Add modules
+		d	Delete modules
+		t	List library table
+		v	Increase verbosity (put before other operation)
+		x	Extract modules
+		V	Print the archiver version
 </verb></tscreen>
 
-You may add modules to a library using the `a' command. If the library
+You may add modules to a library using the `r' command (`a' is deprecated). If the library
```

Comment:
I would expect the 'a' to be mentioned in the list of operations, too, with a remark that it is deprecated.